rakefile.rb

```diff
@@ -47,8 +47,7 @@
     - [ ] attach nupkg
     - [ ] publish the release
 - [ ] push NuGet package
-- [ ] copy release notes from GitHub to NuGet
-- [ ] de-list pre-release or superseded buggy NuGet packages if present (copy any release notes forward to the new version)

```

Comment:
Do we want, then, to leave all x.y.z-beta123 releases on GitHub and rely on them providing the release notes for the production release?
I'd think that we'd want to still consolidate all the notes onto the final release build, as we've done for 2.0.0.